patterns/1-initial/governance-levels.md

```diff
@@ -2,75 +2,76 @@
-  welcome to read the code. They can submit feature requests and bug reports for
-  things they would like to see changed.
+Examples of common InnerSource operating models (1) are:
 
-* **Contributions welcome**: People outside the core development team may use the
-  code, make modifications and feed those modifications back into the projects.
-  Trusted committers are willing to mentor those contributions to a state where
-  they can be accepted or communicate clearly why the proposed change cannot be
-  made.
+- **Bug Reports and Issues Welcome**: People outside the core development team may use the code. They can submit feature requests and bug reports for things they would like to see changed.
+- **Contributions Welcome**: People outside the core development team may use the code. They can also make modifications and submit them to core team for inclusion.
+- **Shared Ownership**: Members of different teams collaborate on the project as equal peers. Everyone has the ability to merge code. Everyone has an equal say on the project direction. Everyone has an equal say in who else to add to this group.
 
-* **Shared write access**: In addition to the above people outside the core
-  development team may gain write access to the source repository. Influence on
-  roadmap decisions as well as influence on who else gains write access is
-  restricted to the core development team.
+Examples of promoting the names (3) are:
 
-* **Shared ownership**: Members of different teams collaborate on the project as
-  equal peers. Everyone has the ability to merge code. Everyone has an equal say
-  on the project direction. Everyone has an equal say in who else to add to this
-  group.
+- Using the names within project documentation and contributing guides (see also [Standard Base Documentation](../2-structured/project-setup/base-documentation.md)).
+- Labelling projects with the names in an [InnerSource Portal](../2-structured/innersource-portal.md).
+- Presenting the names as a menu of adoption options for new InnerSource projects.
+- Including the names and models in any internal InnerSource training or promotion.
 
 ## Resulting Context
 
-Teams can adopt InnerSource best practices in a step-by-step way. By documenting
-individual steps contributor confusion is avoided.
+- Cross team communication occurs efficiently without confusion using terms that are universally understood and centrally documented.
+- Organisation leaders understand the nuances within practising InnerSource and make better informed and more precise decisions that are easier to communicate.
+- Increased standardisation of InnerSource practices within the organisation as the named and documented building blocks are used by teams as a menu for adoption.
+- Teams can adopt InnerSource best practices in a step-by-step way which makes adoption easier and less intimidating.
 
 ## Known Instances
 
-TBD
+<img alt="Flutter InnerSource Pyramid" src="/assets/img/flutter-pyramid.svg" width="70%">
```

Comment:
Note: We need to check whether SVG gets displayed correctly when publishing to our book (on gitbook.com). If not, then we can use PNG instead.

Reply:
Note to self: We will also need markdown notation for the image, in order for gitbook to pick it up correctly.

Reply:
```suggestion
![InnerSource Pyramid used by Flutter Entertainment](../../assets/img/flutter-pyramid.svg)
```